### ISSUE TITLE
Accessibility Support

### ISSUE THREAD

**Issue by fko-kuptec** [DoctorinaAI/md#18]
First of all, thanks for creating a markdown renderer focused on performance. I appreciate! 😊

**Is your feature request related to a problem? Please describe.**
Unfortunately, I cannot use this package with its current feature-set for rendering markdown in our app, since it does not seem to provide semantic information to screen readers. For a production app, however, this is a necessity in my point of view.

**Describe the solution you'd like**
I think the ideal solution would be to provide semantic information for the specific contents currently visible on screen, splitted into headings, paragraphs, etc. The simpler fallback solution would be to show one semantic block for the whole markdown document, like `gpt_markdown` seems to do it. 

I browsed the source code for a few minutes in the hope of being able to add that functionality quickly. However, implementing that feature does seem non-trivial to me, at least in its ideal form.
